### PR TITLE
ddl: refine some error messages related to generated column and expression index

### PR DESCRIPTION
**Pull request by wjhuang2016**
### What problem does this PR solve?

Refine some error messages related to generated column and expression index

### What is changed and how it works?

Proposal: [xxx](url) 

Refine some error messages related to generated column and expression index.

### Check List 

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
'None' 
```

### CODE REVIEW

File: errors.toml [hunk 1]
```diff
@@ -351,11 +351,6 @@ error = '''
 A primary key index cannot be invisible
 '''
 
-["ddl:3753"]
-error = '''
-Cannot create an expression index on a function that returns a JSON or GEOMETRY value
```
Comment (AilinKid) [inline errors.toml:356]: why remove it here?

Comment (wjhuang2016) [reply errors.toml:356]: After I changing it to internal, `make errdoc` will do it.

File: go.mod [hunk 2]
```diff
@@ -78,6 +78,7 @@ require (
 	google.golang.org/grpc v1.27.1
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/yaml.v2 v2.4.0
+	honnef.co/go/tools v0.2.0 // indirect
```
Comment (xiongjiwei) [inline go.mod:81]: where it comes from, I did not see any new imports

Comment (wjhuang2016) [reply go.mod:81]: `Make check`？

**Review by AilinKid — APPROVED** [review]

**Review by xiongjiwei — APPROVED** [review]

**Pull request merged**